src/pybind/mgr/orchestrator/module.py

```diff
@@ -814,24 +814,24 @@ def _daemon_add_osd(self, svc_arg: Optional[str] = None) -> HandleCommandResult:
 
     @_cli_write_command('orch osd rm')
     def _osd_rm_start(self,
-                      svc_id: List[str],
+                      osd_id: List[str],
```

Comment:
Nit, but can we pluralize this parameter so it's "osd_ids" (here and other places it's being used)? I think having <osd_id> in the help output gives the impression you can only provide one. "osd_ids" also matches with what we call the argument in https://github.com/ceph/ceph/blob/master/src/pybind/mgr/cephadm/module.py#L2496

Reply:
mmm. not sure of this is a good idea. I am using the general rule for list parameters. The help for this command is generated automatically.. an it seems that if your a re using a list of parameters, the help automatically adds "..." like in:
`orch osd rm <osd_id>... [--replace] [--force]                  Remove OSD daemons`

So if we pluralize...maybe we need to review all other list parameters of all commands to follow the same rule... or change the command help generation.

Reply:
right. `orch osd rm --osd-id 1 --osd-id 2 --osd-id 3 ...` I think this is ok

Reply:
okay, that makes sense